lib/wallet-account.js

```diff
@@ -164,6 +164,8 @@ class WalletConnection {
         currentUrl.searchParams.delete('public_key');
         currentUrl.searchParams.delete('all_keys');
         currentUrl.searchParams.delete('account_id');
+        currentUrl.searchParams.delete('meta');
+        currentUrl.searchParams.delete('transactionHashes');
```

Comment:
just double checking, is this change intentionally included in this PR? didn't see it mentioned in commit message or comments

Reply:
This is because the last PR didn't commit the JS artifacts.  So this adds them, but generally you can ignore the JS files since they get built before a release anyway.

Reply:
gotcha. thanks!